utils/config.py

```diff
@@ -15,17 +34,19 @@ def get_config(key : str) -> Union[str, list]:
         The data from config_data.json.
     '''
 
-    base_path = (Path(os.path.abspath(__file__)) /'..' /'..').resolve()
-    config_path = base_path / 'config_data.json'
-    with open(config_path) as json_file: # add path thing
-        config_data = json.load(json_file)
+    # TODO: We obtain the data from BotConfig
```

Comment:
Before merging, get_config and update_config in `utils/config.py` should be updated to use BotConfig. This can be something to work on in the next cycle of assignments.